taiga_halo2/examples/simple_sudoku/vp.rs

```diff
@@ -147,15 +147,15 @@ mod tests {
         let app_data_dynamic = pallas::Base::zero();
 
         let value: u64 = 0;
-        let nk_com = NullifierKeyCom::default();
+        let nk = NullifierKey::random(&mut rng);
         let rseed = RandomSeed::random(&mut rng);
         let rho = Nullifier::new(pallas::Base::random(&mut rng));
         Note::new(
             vp_vk,
             app_data_static,
             app_data_dynamic,
             value,
-            nk_com,
+            nk,
```

Comment:
Are we getting rid of the commitment over the nullifier key?

Reply:
What if we call it `NullifierKey::Com`?
I use this new struct to ignore the nullifier_key commitment scheme in case we want to change it. The new struct defines the nk and nk commitment as a `pallas::base` element.